test/framework/filetools.py

```diff
@@ -1724,6 +1724,32 @@ def test_search_file(self):
         for pattern in ['*foo', '(foo', ')foo', 'foo)', 'foo(']:
             self.assertErrorRegex(EasyBuildError, "Invalid search query", ft.search_file, [test_ecs], pattern)
 
+    def test_is_empty_folder(self):
+        tmpdir = tempfile.mkdtemp()
```

Comment:
@Flamefire There's a temporary directory unique to every test already available in `self.test_prefix`, you can use that one as a sandbox (it's also cleaned up automatically).

Reply:
Ok, thanks. Done and improved the readability of the test